ci-operator/step-registry/baremetal/lab/agent/install/ephemeral/gen-ignition/baremetal-lab-agent-install-ephemeral-gen-ignition-ref.yaml

```
@@ -39,6 +39,17 @@ ref:
     default: "/alabama/.cache/agent/image_cache/coreos-x86_64.iso"
   - name: COREOS_IMAGE_NAME
     default: "coreos-x86_64.iso"
+  - name: ADDITIONAL_WORKERS_DAY2
+    default: "true"
+    documentation: |
+      Whether to add (power on) the additional workers after the cluster is installed. And the default value is "true"
+      when ADDITIONAL_WORKERS > 0, "false" is not supported in ABI. This is useful for testing multi-arch compute node 
+      clusters as a real day2 operation and is especially important for testing single-architectures clusters based on
+      single-architecture payloads migrating to a multi-arch payload. Also, at time of writing, the above path is the
+      only one possible for multi-arch clusters with arm64 control plane and additional amd64 workers ( because the 
+      release image is not a manifest list when installing from Prow and this check would fail at installation time if
+      installing an arm64 CP cluster from the multi payload: 
+      https://github.com/openshift/installer/blob/4daff6117fce1d73774b02caf605d9e1b836dd7c/data/data/bootstrap/files/usr/local/bin/release-image-download.sh.template#L36-L50).
```

```suggestion
      Whether to add additional workers after the cluster is installed. And the default value is "true"
      when ADDITIONAL_WORKERS > 0, "false" is not supported in ABI. This is useful for testing multi-arch compute node 
      clusters as a real day2 operation and is especially important for testing single-architectures clusters based on
      single-architecture payloads migrating to a multi-arch payload.
```

That last sentence doesn't apply anymore

---

Got it!

---

@aleskandro Would I remove this sentence of 'baremetal-lab-upi-install-ref' and 'upi-install-heterogeneous-ref' too?

---

if you could do it, that'd be good too!

---

Okay, done!